packages/serverpod/lib/src/server/session.dart

```diff
@@ -290,6 +290,31 @@ class MethodCallSession extends Session {
   }
 }
 
+/// When a call is made to an endpoint method that uses a stream [Server] a
+/// [StreamingMethodCallSession] object is created. It contains all data
+/// associated with the current connection and provides easy access to the
+/// database.
+class StreamingMethodCallSession extends Session {
```

Comment:
Question: What is the thinking with this type? Why not just use the StreamSession ?

Reply:
MethodStreamSession